.travis.yml

```diff
@@ -109,5 +107,7 @@ stages:
   - test
   - name: cd
```

Comment:
Does this need to be defined as a stage somewhere?

Reply:
It is on: https://github.com/whamcloud/integrated-manager-for-lustre/pull/958/files#diff-354f30a63fb0907d4ad57269548329e3R71

just hidden by default by the github review UI